tests/conditional_compilation/conftest.py

```diff
@@ -171,3 +190,85 @@ def save_session_info(pytestconfig, artifacts):
     """Fixture function for saving additional attributes to configuration file."""
     yield
     write_session_info(path=artifacts / SESSION_INFO_FILE, data=pytestconfig.session_info)
+
+
+@pytest.fixture(scope="session")
+def omz_path(request):
+    """Fixture function for command-line option."""
+    omz_path = request.config.getoption("omz_repo", skip=True)
+    validate_path_arg(omz_path, is_dir=True)
```

Comment:
will `omz_path` fixture be called if no omz models provided in config? If yes, and if `--omz_path` wasn't specified, it will lead to crash